tutor/templates/build/openedx/Dockerfile

```diff
@@ -51,8 +55,10 @@ RUN git config --global user.email "tutor@overhang.io" \
 {{ patch("openedx-dockerfile-git-patches-default") }}
 {%- elif EDX_PLATFORM_VERSION == "master" %}
 # Patches in nightly node
+RUN curl -fsSL https://github.com/openedx/edx-platform/commit/3642cab3ac61ddfd360e7ceb7463b52be8c4deb0.patch | git am
```

Comment:
why is this patch needed?

Reply:
[edx-proctoring-proctortrack is installed as an editable VCS dependency](https://github.com/openedx/edx-platform/blob/ecbedbcf78cd74c907f18d76b77a64e31ccf78fe/requirements/edx/github.in#L92) but uv [doesn't support that](https://github.com/astral-sh/uv/issues/5442). I have this PR https://github.com/openedx/edx-platform/pull/35654 with the same change that is used in that patch.

Reply:
The fact that uv doesn't support editable dependencies is a blocking issue, I think. I suggest we wait until the [upstream issue](https://github.com/astral-sh/uv/issues/5442) is resolved.

Reply:
I think they are not going to implement it in the near future. Note that it refers to editable remote requirements, it works fine with editable requirements in a local path.

AFAIK the only use for an editable remote requirement is that it copies all the files, but you can fix that using the `MANIFEST.in`.

Reply:
I understand that. That means that using `uv` means we will no longer be able to support adding editable dependencies to edx-platform. I'm not sure I'm ready to make that breaking change, as I expect that many people (Axim included) might add editable dependencies there.

Can we consider making an upstream PR to support that feature?

Reply:
> Maybe we can keep standard pip as the default with the possibility to swap to uv if your fork of edx-platform doesn't contain any editable dependency.

That's already the case. People can install uv in their own Docker image and alias `pip='uv pip'`, all with a Tutor plugin.

Reply:
> But that wouldn't work for the RUN commands in the Dockerfile, right?

It would. Install uv with the "openedx-dockerfile-minimal" patch, then `ln -s /usr/bin/uv /usr/bin/pip`. Maybe this will be overwritten in the python-requirements layer, in which case we would have to introduce a new patch statement just above `pip install -r /openedx/edx-platform/requirements/edx/base.txt`. Would you like to introduce such a patch?